### DISCUSSION TITLE
Please add the ability to set the default syntax language

### DISCUSSION THREAD

**Discussion by ghost** [outline/outline#6039] [Ideas]
When creating a code block, either through the GUI directly, or if using markdown "```" the default language selected is "javascript".
Please add an option in the admin area to select the default language when code blocks are first created.

Not all of us want javascript to always be the first selection.

**Comment by tommoor** [comment]
The UI will automatically remember your previous selection

**Comment by ghost** [comment]
That is not the behavior I am experiencing.
I continue to select "powershell" as the language, and every time I create a page, or modify existing pages and add new code blocks they continue to default to "javascript"

**Reply by tommoor** [reply]
Converted to issue https://github.com/outline/outline/issues/6040
